wordpress-mobile/WordPress-Android
# [JP Social] Revert Mastodon client side pop-up

This partially reverts #19242 , which shows a popup for Mastodon redirecting the user to an external browser because of a missing `instance` query parameter. 

The pop-up is being reverted for Mastodon because of the web fix (ref: D122942-code) that returns a form asking for the username/instance instead of returning an error page, which is a better solution and doesn't require the user to leave the app.

## To test:
At the time of opening this PR the web fix was not deployed to production yet so testing the app is not possible, though it should be soon.

1. Install Jetpack and log in
2. Go to the Menu > Social
3. Tap on Mastodon
4. Tap Connect
5. Verify a WebView opens with a form asking for a username
6. Continue the connection flow with a Mastodon account
7. Verify the connection is made successfuly

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
